experimental/uefi/channel/src/lib.rs

```diff
@@ -113,59 +367,51 @@ where
         self.inner.write_all(&length_bytes)?;
         let method_or_status_bytes = frame.method_or_status.to_le_bytes();
         self.inner.write_all(&method_or_status_bytes)?;
+        let spacer_bytes = [0; SPACER_SIZE];
+        self.inner.write_all(&spacer_bytes)?;
+        let flag_bytes: [u8; FLAG_SIZE] = frame.flag.into();
+        self.inner.write_all(&flag_bytes)?;
```

Comment:
Can be done later, but we should consolidate all of this writing into a single write onto the inner channel relatively soon. At the moment this code will send 5 packets for a single frame.

Reply:
Makes sense. Does that apply to both virtio console and vsock though? It seems it'd only be the case for vsock. The initial reason for incremental writes was to avoid reallocating to merge all the frame pieces, but agreed that the cost of that is out-weight by sending multiple packages.